R/ioslides_presentation.R

```diff
@@ -18,6 +18,7 @@ ioslides_presentation <- function(logo = NULL,
                                   keep_md = FALSE,
                                   lib_dir = NULL,
                                   md_extensions = NULL,
+                                  analytics = NULL,
```

Comment:
Maybe have `analytics` right after `mathjax` (and before the new `template` argument).